docs/stable/9-waku-usage.md

```diff
@@ -0,0 +1,346 @@
+
+Status uses [Waku](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md) to provide
+privacy-preserving routing and messaging on top of devP2P. Waku uses topics
+to partition its messages, and these are leveraged for all chat capabilities. In
+the case of public chats, the channel name maps directly to its Waku topic.
+This allows anyone to listen on a single channel.
+
+Additionally, since anyone can receive Waku envelopes, it relies on the
+ability to decrypt messages to decide who is the correct recipient. We do
+however not rely on this property, but instead implement another secure
+transport layer on top of Waku.
+
+## Reason
+
+Provide routing, metadata protection, topic-based multicasting and basic
+encryption properties to support asynchronous chat.
+
+## Terminology
+
+* *Waku node*: an Ethereum node with Waku V1 enabled
+* *Waku network*: a group of Waku nodes connected together through the internet connection and forming a graph
+* *Message*: decrypted Waku message
+* *Offline message*: an archived envelope
+* *Envelope*: encrypted message with metadata like topic and Time-To-Live
+
+## Waku packets
+
+| Packet Name          | Code | References |
+| -------------------- | ---: | --- |
+| Status               |    0 | [Status](#status), [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#status) |
+| Messages             |    1 | [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#messages) |
+| Batch Ack            |   11 | Undocumented. Marked for Deprecation |
+| Message Response     |   12 | [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#message-confirmations-update) |
+| Status Update        |   22 | [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#status-update) |
+| P2P Request Complete |  125 | [4/WAKU-MAILSERVER](https://specs.status.im/spec/4) |
+| P2P Request          |  126 | [4/WAKU-MAILSERVER](https://specs.status.im/spec/4), [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#p2p-request) |
+| P2P Messages         |  127 | [4/WAKU-MAILSERVER](https://specs.status.im/spec/4), [WAKU-1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md#p2p-message) |
+
+## Waku node configuration
+
+If you want to run a Waku node and receive messages from Status clients, it must be properly configured.
+
+Waku's Proof Of Work algorithm is used to deter denial of service and various spam/flood attacks against the Waku network. The sender of a message must perform some work which in this case means processing time. Because Status' main client is a mobile client, this easily leads to battery draining and poor performance of the app itself. Hence, all clients MUST use the following Waku node settings:
+* proof-of-work requirement not larger than `0.002` for payloads less than 50,000 bytes
+* proof-of-work requirement not larger than `0.000002` for payloads greater than or equal to 50,000 bytes
```

Comment:
This change may be superseded by https://github.com/status-im/specs/pull/118/files . @cammellos what do you think? 

For clarification I've included this in the new `9-waku-usage.md` doc, and I can see your change is to the `3-whisper-usage.md`. I have not made any material changes myself to `3-whisper-usage.md`

Reply:
It's probably best to keep them separate, yes, if you merge in this first I will rebase against it, and update `9-waku-usage.md` as well